### PR TITLE
Updating colors

### LINKED ISSUE #729: Color update and adding new ones
## Description
Based on previous discussions about brand colors, today we updated the [Design Library with the new palette](https://www.figma.com/file/l4Mt3dn3Ndtrvrb4aLcwXI/Design-Library?node-id=0%3A72&viewport=-819%2C-1377%2C1.0137752294540405). This change includes:
1. Update of brand colors.
2. New color set: Soft Brand.
3. Update of interaction colors.

### Brand colors
The following colors have been updated with a new hex code to match the AA level of accessibility:
1. **Tomato:** new hex code 
2. **Gold:** new hex code
3. **Forest Green:** new hex code
4. **Dark Slate Blue:** new hex code
5. **BrighterDarkSlateBlue:** eliminated

Screen Shot 2020-10-14 at 18 06 43

### Soft Brand set
This set of colors is a new palette as soft versions of brand colors to be used in backgrounds.

Screen Shot 2020-10-14 at 18 08 07

### Interaction colors
Since the interaction colors are based on the brand ones, this adjustment follows the same lightness and saturation rule of the existing ones, and therefore, they were updated.

Screen Shot 2020-10-14 at 18 10 56

### LINKED ISSUE #737: Interaction colors update and feedback colors correction
### Interaction colors
There was an error with the interaction colors after reviewing and updating the palettes (requested in ticket #729). Also, Vocabulary has a button using one of the blue versions but there was no interaction color.

In this update, all interaction colors have been updated:

1. BrighterTomato: `#E04D03`
2. DarkGold: `#F8CC2C`
3. BrighterForestGreen: `#008B00`
4. BrighterBlue: `#174EBC` (New color in the color set)

Screen Shot 2020-10-19 at 17 20 17

### Feedback colors
In this set, two colors use the same hex code of interaction colors. And we need to update them. These colors are:
* `DarkSuccess` = `ForestGreen`.
* `DarkInfo` = `BrigtherBlue`.

All remained colors have their own hex code and no need to be updated

Screen Shot 2020-10-19 at 17 20 09

### PR DESCRIPTION
**Pull request by rochisha0**
## Fixes

Fixes #729 and Fixes #737 by @panchovm 

## Description

It adds new colors (soft brand colors) and updates the existing ones (interaction and brand)

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [x] I added tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Review by dhruvkb — CHANGES_REQUESTED** [review]
Please consistently use a single line format or a multiline format. Mixing both is not good.

File: packages/vocabulary/src/styles/color.scss [hunk 1]
```diff
@@ -1,12 +1,19 @@
...
-$color-tomato: rgb(226, 54, 0);
-$color-gold: rgb(239, 190, 0);
-$color-forest-green: rgb(0, 131, 0);
+$color-tomato: rgb(209, 69, 0);
+$color-gold: rgb(251, 212, 60);
+$color-forest-green: rgb(0, 128, 0);
 $color-dark-turquoise: rgb(5, 181, 218);
-$color-dark-slate-blue: rgb(60, 92, 153);
+$color-dark-slate-blue: rgb(21, 71, 168);
 $color-brighter-dark-slate-blue: rgb(16, 102, 231); // Also used for interaction
```
Comment (zackkrida) [inline packages/vocabulary/src/styles/color.scss]:
```suggestion
```

Please remove this line!

**Review by zackkrida — CHANGES_REQUESTED** [review]
Almost donee, just need to remove the variable for `$color-brighter-dark-slate-blue`

File: packages/vocabulary/src/styles/color.scss (hunk 1 above)
Comment (rochisha0) [reply packages/vocabulary/src/styles/color.scss]: The color is being used in `button.css`.

File: packages/vocabulary/src/stories/color.stories.mdx [hunk 2]
```diff
@@ -51,18 +56,13 @@ text is placed on them.
...
   <ColorItem
     title="BrighterForestGreen"
     subtitle="$color-brighter-forest-green"
-    colors={["rgb(0, 184, 80)"]}
-  />
-  <ColorItem
-    title="BrighterDarkSlateBlue"
-    subtitle="$color-brighter-dark-slate-blue"
-    colors={["rgb(16, 102, 231)"]}
+    colors={["rgb(0, 168, 3)"]}
```
Comment (dhruvkb) [inline packages/vocabulary/src/stories/color.stories.mdx]: The issue of some colours written in a single line and some not has not been resolved yet.

**Review by dhruvkb — CHANGES_REQUESTED** [review]

File: packages/vocabulary/src/stories/color.stories.mdx (hunk 2 above)
Comment (rochisha0) [reply packages/vocabulary/src/stories/color.stories.mdx]: Done!

File: packages/vocabulary/src/styles/color.scss (hunk 1 above)
Comment (zackkrida) [reply packages/vocabulary/src/styles/color.scss]: Oh, that's right! Thank you for this reminder. I'm going to talk to @panchovm about this.

File: packages/vocabulary/src/stories/color.stories.mdx [hunk 3]
```diff
@@ -48,39 +49,18 @@ text is placed on them.
...
-  <ColorItem
-    title="BrighterForestGreen"
-    subtitle="$color-brighter-forest-green"
-    colors={["rgb(0, 184, 80)"]}
-  />
-  <ColorItem
-    title="BrighterDarkSlateBlue"
-    subtitle="$color-brighter-dark-slate-blue"
-    colors={["rgb(16, 102, 231)"]}
-  />
```
Comment (zackkrida) [inline packages/vocabulary/src/stories/color.stories.mdx]: Please re-add this (and make it one line for @dhruvkb 😄)

**Review by zackkrida — CHANGES_REQUESTED** [review]
Okay, `BrighterDarkSlateBlue` will remain as an _interaction_ color, but be removed from the brand colors. 

@rochisha0 thank your for your attention to detail, which brought this problem to my attention.

File: packages/vocabulary/src/styles/color.scss (hunk 1 above)
Comment (zackkrida) [reply packages/vocabulary/src/styles/color.scss]: @rochisha0 Please move this color to the `interaction` section of colors lower in the file. Thanks!

Comment (rochisha0) [reply packages/vocabulary/src/styles/color.scss]: Done! @zackkrida

File: packages/vocabulary/src/stories/color.stories.mdx (hunk 3 above)
Comment (rochisha0) [reply packages/vocabulary/src/stories/color.stories.mdx]: Done!

**Comment by fcoveram** [comment]
This ticket (#737) is related to changes indicated here

**Comment by rochisha0** [comment]
I have updated the PR as per issue #737

**Review by zackkrida — APPROVED** [review]
✅ 

@dhruvkb I am going to let the inconsistent single-line `ColorItem`s slide, as it is caused by prettier's line width. While we don't _officially_ have prettier enabled yet I am okay with following that rule for now.

**Pull request merged**